prboom2/src/dsda/bsp.h

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright(C) 2023 Brian Koropoff
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//	BSP Analysis
+//
+
+#ifndef DSDA_BSP_H_
+#define DSDA_BSP_H_
+
+#include "r_defs.h"
+
+extern iseg_t* isegs;
+extern unsigned int numisegs;
+extern int* adjacency;
+extern unsigned int numadjacency;
+
+void dsda_AnnotateBSP(void);
```

Comment:
If it's not too much agony, I'd prefer that you split out `bsp/dump` so that the business logic is isolated and succinct.

Reply:
I broke geometry/bsp/bspdump stuff into separate files, should flow better now